### ISSUE TITLE
Problem using along with Require CommonJS Modules Helper

### ISSUE THREAD

**Issue by netcell** [babel/babel-sublime#124]
I am having problem with using package [Require CommonJS Modules Helper](https://github.com/jfromaniello/sublime-node-require) as listed in an issue https://github.com/jfromaniello/sublime-node-require/issues/16 while using **Javascript (Babel)** syntax. The error in the console is as follow
```
Traceback (most recent call last):
File ".\require_node.py", line 77, in write
KeyError: u'source.js.jsx'
```
I think the problem is with commit https://github.com/babel/babel-sublime/commit/273cc6f07c3bf89efb05e103d3974320949d1981.

**Comment by zertosh** [comment]
Yeah, I figured the `scopeName` was going to break stuff. I'm sure a bunch of other things assume that the scope is just `source.js`. I'll revert it now and publish an update.

**Comment by netcell** [comment]
Thanks a lot :smile:

**Comment by zertosh** [comment]
Published as v6.2.1. It should get picked up by Package Control within an hour.
